topo_processor/data/data_transformers/data_transformer_repo.py

```diff
@@ -27,7 +27,6 @@ def transform_data(self, item: Item) -> None:
                 except Exception as e:
                     item.add_error(str(e), transformer.name, e)
                     get_log().warning(f"Data Transform Failed: {e}", transformers=transformer.name)
-                    return False
```

Comment:
This changes the semantics of the method, by running the rest of the method rather than returning early when encountering this exception. Are you sure you want to do that?